src/Uno.UI/UI/Xaml/Controls/Popup/PopupBase.cs

```diff
@@ -30,7 +30,6 @@ public partial class PopupBase : FrameworkElement, IPopup
 
 		public PopupBase()
```

Comment:
Seams that the whole ctor can be removed

Reply:
...